plugin.go

```diff
@@ -38,30 +50,206 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	var port string
 	var proxyClient string
 	var useSwaggerUI bool
+	var foreground bool
+	var all bool
 	var cmd = &cobra.Command{
-		Use:   "web",
-		Short: "Starts web server",
+		Use:       "web",
```

Comment:
- [ ] When web ui starts, link should be printed in terminal as well, for user to be able to open it again if browser tab is closed
- [ ] When web ui is already started and user execute command again, it let user know that ui is already running and display link again

```
➜  ski-platform git:(master) plasmactlui web        
Web running in background with PID: 37089
                                      
➜  ski-platform git:(master) plasmactlui web
Error: web server port 8080 you are trying to use is not available
```